spec/javascripts/app/views/poll_view_spec.js

```diff
@@ -75,4 +75,32 @@ describe("app.views.Poll", function(){
       expect(this.view.$('form').length).toBe(0);
     });
   });
+
+  describe("answer given", function() {
+    it("adds 'users-vote' class to progress bar for the option the user voted for", function() {
+      var answer = this.view.poll.poll_answers[0];
+      this.view.model.set("poll_participation_answer_id", answer.id);
+      expect(this.view.$(".poll_progress_bar.users-vote").length).toBe(1);
+    });
+
+    it("doesn't add 'users-vote' class to progress bar of the options the user didn't vote for", function() {
+      var answer1 = this.view.poll.poll_answers[0];
+      var answer2 = this.view.poll.poll_answers[1];
+      this.view.model.set("poll_participation_answer_id", answer1.id);
+      var progressBarAnswer2 = ".poll_progress_bar[data-answerid='" + answer2.id + "']";
+      expect(this.view.$(progressBarAnswer2).hasClass("users-vote")).toBe(false);
```

Comment:
Using a variable for the selector looks unnecessary to me. You could replace `progressBarAnswer2` by the value set before and still follow the maximum line length rule. (120 chars)